### PR TITLE
Add sidebar to docs/web/api/filereader/filereader

### PR DESCRIPTION
**Pull request by samantharamon**
#### Summary
Add APIRef sidebar.

#### Motivation
Write the Docs 2022 Writing Day project.

#### Supporting details
None.

#### Related issues
N/A

#### Metadata
This PR…

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [X] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/filereader/filereader/index.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ tags:
   - Reference
 browser-compat: api.FileReader.FileReader
 ---
+
+{{APIRef(“File API”))}}
```
Comment (queengooborg) [inline files/en-us/web/api/filereader/filereader/index.md:12]:
```suggestion
{{APIRef("File API"))}}
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thank you for your PR!  It looks like smart quotes were accidentally added, rather than normal quotes:

File: files/en-us/web/api/filereader/filereader/index.md (hunk 1 above)
Comment (samantharamon) [reply files/en-us/web/api/filereader/filereader/index.md:12]: Thanks for catching this!

File: files/en-us/web/api/filereader/filereader/index.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ tags:
   - Reference
 browser-compat: api.FileReader.FileReader
 ---
+
+{{APIRef("File API"))}}
```
Comment (samantharamon) [inline files/en-us/web/api/filereader/filereader/index.md:12]:
```suggestion
{{APIRef("File API")}}
```

**Review by queengooborg — APPROVED** [review]
Thanks for the quick changes (and catching that extra parenthesis), this is LGTM!

**Pull request merged**